blueprints/eslint-config-simplabs/files/.eslintrc.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
   root: true,
-  extends: 'simplabs',
+  extends: [
+    'simplabs',
+    'simplabs/rules/ember',
```

Comment:
yes, but then we would have a conflict between our `ember` config (`simplabs/config/ember`) and the `ember` plugin ruleset config (`simplabs/rules/ember` or maybe soon `simplabs/plugin/ember`)